### PR TITLE
Remove what's new page and link to release notes instead

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
fixes https://github.com/rancher/dashboard/issues/9332 - migrating 'getting started' will require coordination with docs team; this PR only updates the release note links. 

Testing should include -rc and -head versions as well as a regular release

### CODE REVIEW

File: cypress/e2e/tests/pages/home.spec.ts [hunk 1]
```diff
@@ -17,13 +15,12 @@ describe('Home Page', () => {
     HomePagePo.goToAndWaitForGet();
   });
 
-  it('Can navigate to What\'s new page', { tags: ['@adminUser', '@standardUser'] }, () => {
+  it.only('Can navigate to What\'s new page', { tags: ['@adminUser', '@standardUser'] }, () => {
```
Comment (cnotv) [inline cypress/e2e/tests/pages/home.spec.ts:18]: I think we should not use only 😅

File: cypress/e2e/tests/pages/home.spec.ts [hunk 2]
```diff
@@ -17,13 +15,12 @@ describe('Home Page', () => {
...
     /**
-     * Click link on home page and verify user lands on What's new page
-     * Verify contents of What's new page
-     * Verify changlog banner is hidden after navigating to the page
+     * Verify changlog banner is hidden after clicking link
+     * Verify release notes link is valid github page
      */
-    const burgerMenuPo = new BurgerMenuPo();
+    // const burgerMenuPo = new BurgerMenuPo();
```
Comment (cnotv) [inline cypress/e2e/tests/pages/home.spec.ts:23]: we can probably remove this

File: shell/store/index.js [hunk 3]
```diff
@@ -573,6 +573,18 @@ export const getters = {
     return state.targetRoute;
   },
 
+  releaseNotesUrl(state, getters) {
+    const version = getters['management/byId'](MANAGEMENT.SETTING, 'server-version')?.value;
+
+    const base = 'https://github.com/rancher/rancher/releases';
+
+    if (version && version.match(/v[0-9]+\.[0-9]+\.[0-9]$/)) {
```
Comment (cnotv) [inline shell/store/index.js:581]: Is there anything we can get from the util? If not maybe we can extend it?

https://github.com/rancher/dashboard/blob/master/shell/utils/version.js

**Review by cnotv — CHANGES_REQUESTED** [review]

File: cypress/e2e/tests/pages/home.spec.ts (hunk 1 above)
Comment (mantis-toboggan-md) [reply cypress/e2e/tests/pages/home.spec.ts]: Updated to remove only 😅

File: shell/store/index.js (hunk 3 above)
Comment (mantis-toboggan-md) [reply shell/store/index.js]: Good call - I've updated the getter to use the existing function `isDevBuild`

**Review by cnotv — APPROVED** [review]
LGTM

**Pull request merged**